so.libdb.gtkcord4.yml

```diff
@@ -0,0 +1,244 @@
+id: so.libdb.gtkcord4
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: gtkcord4
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.kwalletd5
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+
+modules:
+  - name: gtkcord4
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+    build-commands:
+      - go install -v -mod=vendor ./...
+      - desktop-file-edit --set-icon=$FLATPAK_ID --remove-key=DBusActivatable nix/$FLATPAK_ID.desktop
```

Comment:
`nix/$FLATPAK_ID.desktop` -> `xyz.diamondb.gtkcord4.desktop`

Reply:
the proper fix is to NOT use `$FLATPAK_ID` here. Because inherently the name in he repository is NOT dependent on that value. Only the installed one.